src/tests/receiving/editsInReceiving.test.ts

```diff
@@ -0,0 +1,359 @@
+  }) => {
+    const lot = 'add-lot-test';
+    const expDate = getDateByOffset(new Date(), 5);
+
+    await test.step('Go to stock movement show page', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to shipment receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Open edit modal for item without lot and add lot and exp date', async () => {
+      await receivingPage.receivingStep.table.row(2).editButton.click();
+      await receivingPage.receivingStep.editModal.isLoaded();
+      await receivingPage.receivingStep.editModal.table
+        .row(0)
+        .lotNumberField.textbox.fill(lot);
+      await receivingPage.receivingStep.editModal.table
+        .row(0)
+        .expiryDatePickerField.fill(expDate);
+      await receivingPage.receivingStep.editModal.saveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Assert added lot and exp date on receive page', async () => {
+      await expect(
+        receivingPage.receivingStep.table.getCellValue(2, 'Lot/Serial No.')
+      ).toContainText(lot);
+      await expect(
+        receivingPage.receivingStep.table.getCellValue(2, 'Expiration date')
+      ).toContainText(formatDate(expDate, 'MM/DD/YYYY'));
+    });
+
+    await test.step('Select item to receive', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await receivingPage.receivingStep.table
+        .row(2)
+        .receivingNowField.textbox.fill('10');
+    });
+
+    await test.step('Go to check page and receive shipment', async () => {
+      await receivingPage.nextButton.click();
+      await receivingPage.checkStep.isLoaded();
+      await receivingPage.checkStep.receiveShipmentButton.click();
+      await stockMovementShowPage.isLoaded();
+    });
+  });
+
+  test('Split line into 2 lots', async ({
+    stockMovementShowPage,
+    receivingPage,
+  }) => {
+    const lot = 'add-lot-test';
+    const expDate = getDateByOffset(new Date(), 5);
+
+    await test.step('Go to stock movement show page', async () => {
+      await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to shipment receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Open edit modal for item with lot and split into 2 lots', async () => {
+      await receivingPage.receivingStep.table.row(1).editButton.click();
+      await receivingPage.receivingStep.editModal.isLoaded();
+      await receivingPage.receivingStep.editModal.addLineButton.click();
+      await receivingPage.receivingStep.editModal.table
+        .row(0)
+        .quantityShipped.fill('15');
```

Comment:
Why you cannot use quantityShippedField here? If it is not working and cannot be used here, then it should be removed. `receivingNowField` seems to be working, what is different here?